README.md

```diff
@@ -1,2 +1,18 @@
 # ostis-ps-lib
 OSTIS problem solvers components library
+
+this is draft of a readme
```

Comment:
Why is it draft?

Reply:
it was created just to contain a couple of lines with how-to-launch
I needed to get code review and after that I'll make not draft readme